_docs-v3/agenda-view/agenda-custom-view-demo.html

```diff
@@ -4,9 +4,7 @@
 ---
 <script>
 
-  $(function() {
-
-    $('#calendar').fullCalendar({
+    new Calendar({
```

Comment:
yes, this is good. please proceed with converting the other statements like this